daemon/main.c

```diff
@@ -660,6 +657,26 @@ static void set_nofile_limit(struct rlimit *rl) {
         netdata_log_error("Number of open file descriptors allowed for this process is too low (RLIMIT_NOFILE=%zu)", (size_t)rl->rlim_cur);
 }
 
+static void set_libuv_worker_threads()
+{
+#if defined(ENV32BIT)
+    const int min_libuv_worker_threads = 8;
+    const int max_libuv_worker_threads = 128;
+#else
+    const int min_libuv_worker_threads = 16;
+    const int max_libuv_worker_threads = 1024;
+#endif
+
+    int n = (int) get_netdata_cpus() * 6;
```

Comment:
@vkalintiris to let code more clear, I suggest create a macro with value `6` and use it here, this magic number can complicate understanding.

Reply:
Up for suggestions, the constant seems to be random either way https://github.com/netdata/netdata/blame/1288256c1d68dba448f59d5954a3ce4d74887f04/daemon/main.c#L1836